doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,89 @@
+[id='proc_kogito-deploying-on-ocp-console']
+
+= Deploying {PRODUCT} services on {OPENSHIFT} using the OpenShift web console
+
+After you create your {PRODUCT} services as part of a business application, you can use the {OPENSHIFT} web console to deploy your services. As an aid to deploy your services, you can also use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can check out the project, build it (including native compilation, if applicable), and deploy the resulting services.
+
+.Prerequisites
+* The application with your {PRODUCT} services is in a Git repository.
+* You have access to the OpenShift web console with `cluster-admin` permissions.
+
+.Procedure
+. In the OpenShift web console, go to *Operators* -> *OperatorHub* in the left menu, search for and select *Kogito Operator*, and follow the on-screen instructions to install the latest operator version.
++
+--
+On {OPENSHIFT} 3.11, the OperatorHub catalog is not available by default in the web console, so you must use the `oc` CLI to manually install the {PRODUCT} Operator for your defined project:
+
+.Manually installing {PRODUCT} Operator on OpenShift 3.11
+[source]
+----
+$ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
+$ oc new-project PROJECT_NAME
+$ ./hack/3.11deploy.sh
+----
+--
+. After you install the {PRODUCT} Operator, in the OpenShift web console, go to *Operators* -> *Installed Operators* and select *Kogito Operator*.
+. In the operator page, select the *Kogito Service* tab and click *Create Kogito App* to create the {PRODUCT} service definition.
++
+.Create a {PRODUCT} service definition
+image::kogito/kogito-ocp-create-app.png[Image of Kogito service page in web console]
+. In the application window, drag and drop a YAML or JSON file that contains your service definition, or manually define the service data in the application window.
++
+--
+At a minimum, define the application configurations shown in the following example YAML file:
+
+.Example YAML definition for an application with {PRODUCT} services
+[source,yaml,subs="attributes+"]
+----
+apiVersion: app.kiegroup.org/v1alpha1 # {PRODUCT} API for this service
+kind: KogitoApp  # Application type
+metadata:
+  name: example-quarkus  # Application name
+  namespace: workshop-devconf  # OpenShift project namespace
```

Comment:
I'd change this namespace to `kogito`

Reply:
Done. Will also need update in the screenshots, but sounds like I should wait to update all anyway once the UI settles.